package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "local-home-virual-environment",
+  "version": "0.0.1",
+  "description": "",
+  "private": true,
+  "license": "Apache-2.0",
+  "scripts": {
+    "compile": "tsc",
+    "test": "ava --verbose",
+    "lint": "eslint ."
+  },
+  "dependencies": {
+    "@google/local-home-sdk": "^1.0.0",
+    "cbor": "^5.0.2",
```

Comment:
Those are dependencies of the sample, it shouldn't be needed for the tooling.

Reply:
Will remove.

Reply:
Done